src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/inmemoryrepo/InMemoryCourseRepository.kt

```diff
@@ -67,41 +66,36 @@ class InMemoryCourseRepository : CourseRepository {
 
     private val offerings: Set<Offering> = setOf(
       Offering(
+        id = UUID.randomUUID(),
         organisationId = "MDI",
-        duration = Duration.parse("P10D"),
```

Comment:
Can we remove these in their own branch/PR rather than on the side of this one for adding a database?